test/gradcheck.jl

```diff
@@ -217,6 +217,22 @@ end
   @test gradtest(x -> dropdims(x, dims = (1, 2, 3)), rand(1, 1, 1, 3))
 end
 
+@testset "$f(::AbstractArray)" for f in (real, conj, imag)
+  rng, N = MersenneTwister(123456), 3
+  DTs = (Real, Complex)
+  Ts = (Float64, ComplexF64)
+  @testset "$f(::Array{<:$IDT})" for (IT, IDT) in zip(Ts, DTs)
```

Comment:
I'm a little bit confused by the need for both `Ts` and `DTs`. It looks to me like `DTs` is just for pretty printing. Have I understood correctly?

Reply:
Yes that's right

Reply:
Hmm okay. Would you be happy to remove the `DTs` to remove some visual noise from the test? My feeling is that it just adds visual noise to the tests, and printing `Array{Float64}` is perfectly adequate.

Reply:
Done!